tests/integ/modin/frame/test_apply_axis_0.py

```diff
@@ -88,11 +88,12 @@ def test_axis_0_basic_types_without_type_hints(data, func, return_type):
     # this test processes functions without type hints and invokes the UDTF solution.
     native_df = native_pd.DataFrame(data, columns=["A", "b"])
     snow_df = pd.DataFrame(data, columns=["A", "b"])
+    # np.min is mapped to builtin function so no UDTF is required.
     with SqlCounter(
-        query_count=11,
-        join_count=2,
-        udtf_count=2,
-        high_count_expected=True,
+        query_count=1 if func == np.min else 11,
```

Comment:
This is a huge improvement.

Reply:
Yes, whenever we can avoid the standard apply path, it's a huge win.